src/draw/CesiumDraw.js

```diff
@@ -68,9 +72,29 @@ export class CesiumDraw extends EventTarget {
     if (value) {
       if (!this.eventHandler_) {
         this.eventHandler_ = new ScreenSpaceEventHandler(this.viewer_.canvas);
-        this.eventHandler_.setInputAction(this.onLeftClick_.bind(this), ScreenSpaceEventType.LEFT_CLICK);
+        if (this.entityForEdit) {
+          this.eventHandler_.setInputAction(event => this.onLeftDown_(event), ScreenSpaceEventType.LEFT_DOWN);
+          this.eventHandler_.setInputAction(event => this.onLeftUp_(event), ScreenSpaceEventType.LEFT_UP);
+          if (this.type !== 'point') {
```

Comment:
but there is only one case - not point

Reply:
Your code handles all geometries:
- points (in the else block),
- polygons (including rectangles) with `this.entityForEdit.polygon` and `this.type === 'rectangle'`
- line in the second case of the ternary: `this.entityForEdit.polyline`.

It would help maintenance to have each case clearly defined.